assets/app/view/game/part/track.rb

```diff
@@ -86,12 +88,12 @@ def index_for(path)
           end
         end
 
-        def color_for_index(index)
-          index ? self.class::ROUTE_COLORS[index] : TRACK_COLOR
+        def prop_for_index(index, prop)
+          index ? setting_for("r#{index}_#{prop}") : TRACK[prop]
         end
 
-        def color_for(path)
-          color_for_index(index_for(path))
+        def prop_for(path, prop)
```

Comment:
change all of these to props

Reply:
?

Reply:
%s/prop/props/g

Reply:
on second thought: `value_for(path, prop)` might be more adequate
yep still prop, because it asks for _a_ property, not multiple properties

Reply:
sounds good